### DISCUSSION TITLE
Show/hide multiple ranges of rows/columns

### DISCUSSION THREAD

**Discussion by simonante** [andmarti1424/sc-im#843] [Q&A]
I can :hiderow :showrow a single range, e.g. :hiderow 35:40.
How to hide/show multiple ranges at once, e.g. 35:40, 50:55, 60:65 ?

**Comment by andmarti1424** [comment]
You cannot do it directly inside the ui. but you can do it outside if you pipe something like this:
`echo -e "HIDEROW 6:8\nHIDEROW 11:13" | ./sc-im a.sc`
